libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala

```diff
@@ -162,13 +162,28 @@ object Messages {
     def deviceUuid: DeviceId
   }
 
+  final case class DeviceUpdateAssignmentRequested(
+      namespace: Namespace,
+      eventTime: Instant,
+      correlationId: CorrelationId,
+      deviceUuid: DeviceId,
+      sourceUpdateId: String
```

Comment:
Why is it not of `UpdateId` type as in `AssignUpdateRequest` of Director?

Reply:
It could be OEM-specific ID in an OEM-specific implementation of director as in Sachertorte.

Reply:
It could be OEM-specific ID in an OEM-specific implementation of director as in Sachertorte.

Reply:
Thanks, got it. Shall we then use an AnyVal class-wrapper for type-safety?

Reply:
Probably would not harm...

Reply:
Done in #104